server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSOptionsCheckBaseUtility.java

```diff
@@ -565,4 +565,15 @@ public <E extends ParserRuleContext> void checkBrowsingCommon(E ctx, int coreTok
             checkBrowsingHasNotParameter(ctx, coreToken);
         } else checkStatementHasParameter(ctx, coreToken);
     }
+
+    protected void checkHasTooManyOptions(ParserRuleContext parentCtx, int maxOptions) {
+        List<ParseTree> commandOoptions = parentCtx.children.stream()
+                .filter(node -> node instanceof TerminalNode)
+                .filter(node -> !node.getText().equals("AID"))
+                .collect(Collectors.toList());
+        if (commandOoptions.size() > maxOptions) {
+            throwException(
+                    ErrorSeverity.ERROR, getLocality(parentCtx), "Too many options provided for: ", "HANDLE AID");
+        }
+    }
```

Comment:
Given the dependency on `AID` keyword and the hardcoded message, this should be move to `CICSHandleOptionsCheckUtility.java`.

Also does it work correctly if you specify `aid` (lowercase)?